.idea/.gitignore

```diff
@@ -0,0 +1,2 @@
+# Default ignored files
```

Comment:
Please do not check in your IDE byproducts. See the section [here](https://help.github.com/en/articles/ignoring-files) about global gitignore files for that topic